src/main/java/org/openlmis/notification/repository/EmailVerificationTokenRepository.java

```diff
@@ -27,4 +32,10 @@ public interface EmailVerificationTokenRepository
 
   EmailVerificationToken findOneByEmailAddress(String emailAddress);
 
+  @Modifying
+  @Transactional
```

Comment:
Why it needs `@Transactional`? Generally, we should keep `@Transactional` limited to service layer only.